pages/getting-started/install-memgraph/amazon-linux.mdx

```diff
@@ -0,0 +1,166 @@
+
+Install and run Memgraph from a RPM package on: 
+  - **Amazon Linux 2**
+
+  <Callout type="warning"> 
+Newer versions of Memgraph might not be backward
+compatible with older versions and you might not be able to load storage
+snapshots between different versions. In that case, please contact the Memgraph
+team and we'll help with the migration as much as possible.
+ </Callout>
+
+  ## Installation guide
+
+  <Steps>
+
+  {<h3>Download Memgraph RPM Package</h3>}
+
+Downloaded the latest Memgraph RPM Package from the [Memgraph Download
+Hub](https://memgraph.com/download/).
+
+You can also use [direct download
+links](/getting-started/install-memgraph/direct-download-links) to get the
+latest Memgraph packages. 
+
+For example for Amazon Linux 2:
+
+<CodeSnippet code="sudo wget https://download.memgraph.com/memgraph/v2.15.1/amzn-2/memgraph-2.15.1_1-1.x86_64.rpm" page="install-memgraph/amazon-linux" os="amazon-linux"/>
+
+{<h3>Install Memgraph</h3>}
+
+After downloading the Memgraph RPM package, you can install it by issuing the
+following command:
+
+```console
+sudo yum install ./memgraph-2.15.1_1-1.x86_64.rpm
+```
+
+If you run into any troubles while installing Memgraph, contact us on
+[Discord](https://discord.gg/memgraph).
+
+
+{<h3>Start Memgraph</h3>}
+
+After successful installation, Memgraph can be started as a service using the
+following command:
+
+```console
+sudo systemctl start memgraph
+```
+To verify that Memgraph is running, run the following command:
+
+```console
+journalctl --unit memgraph
+```
+
+If successful, you should receive an output similar to the following:
+
+```console
+You are running Memgraph vX.X.X
+```
+
+If you want the Memgraph database instance to start automatically on each startup, run the
+following command:
+
+```console
+systemctl enable memgraph
+```
+
+You can also check the status of the Memgraph database instance by running:
+```console
+systemctl status memgraph
+```
+
+{<h4>Troubleshooting</h4>}
+
+When you start a Memgraph instance, you may see the following list of notes in
+your terminal:
+
+```console
+You are running Memgraph v1.4.0-community
+
+NOTE: Please install networkx to be able to use graph_analyzer module. Using Python:
+3.8.2 (default, Jul 16 2020, 14:00:26)
+[GCC 9.3.0]
+
+NOTE: Please install networkx to be able to use Memgraph NetworkX wrappers. Using Python:
+3.8.2 (default, Jul 16 2020, 14:00:26)
+[GCC 9.3.0]
+
+NOTE: Please install networkx, numpy, scipy to be able to use proxied NetworkX algorithms. E.g., CALL nxalg.pagerank(...).
+Using Python:
+3.8.2 (default, Jul 16 2020, 14:00:26)
+[GCC 9.3.0]
+
+NOTE: Please install networkx to be able to use wcc module.
+Using Python:
+3.8.2 (default, Jul 16 2020, 14:00:26)
+[GCC 9.3.0]
+```
+
+If you wish to work with built-in NetworkX modules in Memgraph, you need to
+install the following Python libraries:
+* [NumPy](https://numpy.org/)
+* [SciPy](https://www.scipy.org/)
+* [NetworkX](https://networkx.org/)
+
```

Comment:
Is this still true? I don't have Linux, so just checking here. I guess it's a bit different in Docker

Reply:
Done.